woocommerce-gateway-ebanx/assets/js/advanced-options.js

```diff
@@ -76,6 +76,17 @@
       if (colombiaVal != null && colombiaVal.length > 0) {
         enableFields(fields.filter('.ebanx-colombia-document'));
       }
+
+      if (brazilVal == null && chileVal == null && colombiaVal == null) {
+        document.getElementById('woocommerce_ebanx-global_advanced_options_title').style.display = 'none';
+        disableFields(ebanxAdvancedOptionEnable);
+      }
+
+      else {
+        document.getElementById('woocommerce_ebanx-global_advanced_options_title').style.display = 'table';
```

Comment:
It's almost the same:

We can change it to jQuery, because all of our code is using it. The new code can be: 

```
$('#woocommerce_ebanx-global_advanced_options_title').css('display', 'table');
```